incubator/redis-auto-cluster/values.yaml

```diff
@@ -0,0 +1,53 @@
+nameOverride: ""
+serviceNameOverride: ""
+
+shards:
+  count: 3
+  replicationFactor: 1
+  sizeGb: 1
+
+nodeSelector: {}
+
+tolerations: []
+
+persistence:
+  storageClassName: standard
+
+antiAffinity: false
+
+image:
+  repository: redis
+  tag: 4.0.9-alpine
+  pullPolicy: IfNotPresent
+
+config:
+  appendOnly: "yes"
+  clusterNodeTimeout: 15000
+  maxMemoryPolicy: "noeviction"
+  saveOptions: |
+    save 900 1
+    save 300 10
+    save 60 10000
+
+autoCluster:
+  enabled: true
+  image:
+    repository: uenicontrib/redis-auto-cluster-builder
+    tag: 1.0.0
+    pullPolicy: IfNotPresent
+
+security:
+  enabled: false
+
+tuning: false
+
+metrics:
+  enabled: false
+  image:
+    repository: oliver006/redis_exporter
+    tag: v0.17.2
+    pullPolicy: IfNotPresent
+  resources: {}
+  annotations:
```

Comment:
This is not used. However, it'd be better to make annotations generally configurable and remove the hard-coded ones from the service. Defaults should be commented out. You don't need them if you use the Prometheus Operator.